tests/test_brownian_path.py

```diff
@@ -97,3 +97,24 @@ def test_normality(brownian_class, device):
 
         _, pval = kstest(samples_, ref_dist.cdf)
         assert pval >= ALPHA
+
+
+@pytest.mark.parametrize("device", devices)
+@pytest.mark.parametrize("random_order", [False, True])
+def test_continuity(device, random_order):
+    if device == gpu and not torch.cuda.is_available():
+        pytest.skip(msg="CUDA not available.")
+
+    ts = torch.linspace(0., 1., 10000, device=device)
+    bm = torchsde.BrownianPath(t0=ts[0], t1=ts[-1], w0=torch.randn((), dtype=ts.dtype, device=device))
+    vals = torch.empty_like(ts)
+    i_ = torch.arange(len(ts), device=device)
```

Comment:
This could also be simplified as described above.

Reply:
Yup.